arch/arm/boot/dts/overlays/revpi-hat-eeprom-overlay.dts

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright 2022 KUNBUS GmbH
+ */
+
+/dts-v1/;
+/plugin/;
+
+/{
+	compatible = "brcm,bcm2835";
+
+	fragment@0 {
+		target = <&i2c0if>;
```

Comment:
Typo: i2c0if -> i2c0

Reply:
No i2c0if is acctually correct. As the i2c0 is shared with the video core, this node is rather strange.
There is the i2c0if node:
https://github.com/RevolutionPi/linux/blob/a80991905191b94e5f5128d45e17a2fec700c6af/arch/arm/boot/dts/bcm283x.dtsi#L339
And the i2c0mux node:
https://github.com/RevolutionPi/linux/blob/a80991905191b94e5f5128d45e17a2fec700c6af/arch/arm/boot/dts/bcm283x.dtsi#L349
The i2c0mux node has a i2c0 node:
https://github.com/RevolutionPi/linux/blob/a80991905191b94e5f5128d45e17a2fec700c6af/arch/arm/boot/dts/bcm283x.dtsi#L360
But the i2c0 node for the i2cmux node is only needed when the i2c is shared with the camera interface. At least this is how I understand this.

LT;DR:
I used the i2c0 overlay as base for this overlay:
https://github.com/RevolutionPi/linux/blob/revpi-5.10/arch/arm/boot/dts/overlays/i2c0-overlay.dts

Reply:
Interesting, thx for the explanation!